dev/PowerNotifications/PowerNotifications.h

```diff
@@ -592,6 +605,41 @@ namespace winrt::Microsoft::ProjectReunion::factory_implementation
             RaiseEvent(systemAwayModeStatusFunc);
         }
 
+        //SystemSuspend Functions
+        winrt::Microsoft::ProjectReunion::SystemSuspendStatus SystemSuspendStatus()
+        {
+            return static_cast<winrt::Microsoft::ProjectReunion::SystemSuspendStatus>(m_systemSuspendStatus);
```

Comment:
What happens if somebody calls this before anyone has registered a listener and had it fire? Isn't this undefined?
Is the expectation that this is only legal to call from inside of a callback? If so, that's different then other propgets on the same class, which seems potentially confusing.

Reply:
Added uninitialized enum value to handle that. Will be finalized by Andrew/Jon as discussed.